### ISSUE TITLE
quitting without message

### ISSUE THREAD

**Issue by natedogith1** [ansible/ansible-lint#2047]
##### Summary

ansible-lint quitting without any message. It looks like the cause is [raise SystemExit in utils.find_children](https://github.com/ansible/ansible-lint/blob/61ed3eaeb99872c95ea5f8c7d391fcb7ea11dce8/src/ansiblelint/utils.py#L191)

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible [core 2.12.3]
  config file = None
  configured module search path = ['/home/nfaulkne/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nfaulkne/.local/lib/python3.8/site-packages/ansible
  ansible collection location = /home/nfaulkne/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/nfaulkne/.local/bin/ansible
  python version = 3.8.10 (default, Nov 26 2021, 20:14:08) [GCC 9.3.0]
  jinja version = 2.10.1
  libyaml = True
ansible-lint --version
ansible-lint 6.0.2 using ansible 2.12.3
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

WSL Ubuntu

##### STEPS TO REPRODUCE

playbook.yml:
```yaml
- name: task
  hosts: all
  roles:
    - foo
```
roles/foo/tasks/file.yml:
```yaml
- name: one
 - name: two # bad identation
```

```console (paste below)
ansible-lint playbook.yml
```

##### Desired Behavior

An error message should be printed

##### Actual Behavior

ansible-lint exits without showing any message.

```paste below
$ ansible-lint playbook.yml  -vvv
WARNING: PATH altered to include /usr/bin
DEBUG    Logging initialized to level 10
DEBUG    Options: Namespace(cache_dir='/home/nfaulkne/.cache/ansible-compat/0cb87f', colored=True, config_file=None, configured=True, cwd=PosixPath('/home/nfaulkne/ansible'), display_relative_path=True, enable_list=[], exclude_paths=['.cache', '.git', '.hg', '.svn', '.tox'], extra_vars=None, format='rich', kinds=[{'jinja2': '**/*.j2'}, {'jinja2': '**/*.j2.*'}, {'text': '**/templates/**/*.*'}, {'inventory': '**/inventory/**.yml'}, {'requirements': '**/meta/requirements.yml'}, {'galaxy': '**/galaxy.yml'}, {'reno': '**/releasenotes/*/*.{yaml,yml}'}, {'playbook': '**/playbooks/*.{yml,yaml}'}, {'playbook': '**/*playbook*.{yml,yaml}'}, {'role': '**/roles/*/'}, {'tasks': '**/tasks/**/*.{yaml,yml}'}, {'handlers': '**/handlers/*.{yaml,yml}'}, {'vars': '**/{host_vars,group_vars,vars,defaults}/**/*.{yaml,yml}'}, {'meta': '**/meta/main.{yaml,yml}'}, {'yaml': '.config/molecule/config.{yaml,yml}'}, {'requirements': '**/molecule/*/{collections,requirements}.{yaml,yml}'}, {'yaml': '**/molecule/*/{base,molecule}.{yaml,yml}'}, {'requirements': '**/requirements.yml'}, {'playbook': '**/molecule/*/*.{yaml,yml}'}, {'yaml': '**/{.ansible-lint,.yamllint}'}, {'yaml': '**/*.{yaml,yml}'}, {'yaml': '**/.*.{yaml,yml}'}], lintables=['playbook.yml'], listrules=False, listtags=False, loop_var_prefix=None, mock_modules=[], mock_roles=[], offline=None, parseable=False, progressive=False, project_dir='.', quiet=0, rules={}, rulesdir=[], rulesdirs=['/home/nfaulkne/.local/lib/python3.8/site-packages/ansiblelint/rules'], skip_action_validation=True, skip_list=[], tags=[], use_default_rules=False, var_naming_pattern=None, verbosity=3, version=False, warn_list=['experimental', 'role-name'], write=False)
DEBUG    /home/nfaulkne/ansible
INFO     Set ANSIBLE_LIBRARY=/home/nfaulkne/.cache/ansible-compat/0cb87f/modules:/home/nfaulkne/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/nfaulkne/.cache/ansible-compat/0cb87f/collections:/home/nfaulkne/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/home/nfaulkne/.cache/ansible-compat/0cb87f/roles:roles:/home/nfaulkne/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
DEBUG    Loading rules from /home/nfaulkne/.local/lib/python3.8/site-packages/ansiblelint/rules
DEBUG    Effective yamllint rules used: {'braces': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 0, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'brackets': {'level': 'error', 'forbid': False, 'min-spaces-inside': 0, 'max-spaces-inside': 0, 'min-spaces-inside-empty': -1, 'max-spaces-inside-empty': -1}, 'colons': {'level': 'error', 'max-spaces-before': 0, 'max-spaces-after': 1}, 'commas': {'level': 'error', 'max-spaces-before': 0, 'min-spaces-after': 1, 'max-spaces-after': 1}, 'comments': {'level': 'warning', 'require-starting-space': True, 'ignore-shebangs': True, 'min-spaces-from-content': 1}, 'comments-indentation': False, 'document-end': False, 'document-start': False, 'empty-lines': {'level': 'error', 'max': 2, 'max-start': 0, 'max-end': 0}, 'empty-values': False, 'hyphens': {'level': 'error', 'max-spaces-after': 1}, 'indentation': {'level': 'error', 'spaces': 'consistent', 'indent-sequences': True, 'check-multi-line-strings': False}, 'key-duplicates': {'level': 'error'}, 'key-ordering': False, 'line-length': {'level': 'error', 'max': 160, 'allow-non-breakable-words': True, 'allow-non-breakable-inline-mappings': False}, 'new-line-at-end-of-file': {'level': 'error'}, 'new-lines': {'level': 'error', 'type': 'unix'}, 'octal-values': False, 'quoted-strings': False, 'trailing-spaces': {'level': 'error'}, 'truthy': {'level': 'warning', 'allowed-values': ['true', 'false'], 'check-keys': True}}
INFO     Executing syntax check on playbook.yml (0.46s)
```

**Comment by ziegenberg** [comment]
I might be wrong, but as far as I know, this is on purpose, as you do not have a `roles/foo/tasks/main.yml` that includes the `roles/foo/tasks/file.yml` task file. `ansible-lint` does not lint any `yml` file it comes across, but only "lintables" that would be included in a playbook run.
